components/ImprovedSearch/Hit.tsx

```diff
@@ -109,7 +113,7 @@ export const Hit = ({ hit, hasQuery }: HitProps) => {
         />
       </Typography>
       {hasQuery ? (
-        <StyledTypography variant="body_short">
+        <StyledTypography variant="body_short" style={{ fontSize: "0.87rem", fontWeight: "regular" }}>
```

Comment:
Same here, this is a really weird font size? `13.92px`. And did you agree upon that the font size should change only if there's a query? 🤔

Reply:
In both actually ..I am updating

Reply:
Looks good! Had a live demo with Espen and we agreed upon that 14px was a little bit too small so I'm changing it back to 16